sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/SearchIndexerSkillsetConverter.java

```diff
@@ -69,8 +71,13 @@ public static SearchIndexerSkillset map(com.azure.search.documents.indexes.imple
         String description = obj.getDescription();
         searchIndexerSkillset.setDescription(description);
 
-        String eTag = obj.getETag();
-        searchIndexerSkillset.setETag(eTag);
+        searchIndexerSkillset.setDescription(obj.getDescription());
```

Comment:
does these setDescription() call twice?

Reply:
Two different methods, one converts from the codegen type to hand-written and the other from hand-written to codegen.